dotnet-test-samples/kinesis-lambda-dynamodb/kinesis-lambda-dynamodb.sln

```diff
@@ -0,0 +1,59 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{D5C323E6-0454-4A25-B2B0-9C8296D351F6}"
+	ProjectSection(SolutionItems) = preProject
+		README.md = README.md
```

Comment:
I don't think we should have the readme as part of the solution since we did not do that in previous samples - what do you think?